tests/workflows/test_workflow1.py

```diff
@@ -0,0 +1,60 @@
+import os
+from argparse import Namespace
+from tempfile import TemporaryDirectory
+
+import pandas as pd
+
+from mawpy.constants import (USER_ID, UNIX_START_T, UNIX_START_DATE, ORIG_LAT, ORIG_LONG, ORIG_UNC, STAY_LAT, STAY_LONG,
+                             STAY_UNC, STAY, STAY_DUR)
+from mawpy.steps.incremental_clustering import incremental_clustering
+from mawpy.steps.update_stay_duration import update_stay_duration
+from mawpy.workflows.workflow1 import workflow1
+
+
+def test_workflow1_ic_usd():
+    print(os.getcwd())
+    with TemporaryDirectory() as tempdir:
+        input_file = os.path.dirname(__file__) + '/../resources/test_input.csv'
+        output_file = os.path.join(tempdir, 'test_output_workflow1.csv')
+
+        # Prepare arguments similar to command-line arguments
```

Comment:
why is this needed? you could just pass the variables

Reply:
Can you please elaborate a bit more?

do you mean directly passing
os.path.dirname(__file__) + '/../resources/test_input.csv'
os.path.join(tempdir, 'test_output_workflow1.csv')